hudi-client/hudi-client-common/src/test/java/org/apache/hudi/index/bucket/TestBucketIdentifier.java

```diff
@@ -26,13 +26,15 @@
 import org.apache.avro.Schema;
 import org.apache.avro.generic.GenericData;
 import org.apache.avro.generic.GenericRecord;
+import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 
 import java.util.Arrays;
 import java.util.List;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
+@Tag("functional")
```

Comment:
Could you double-check if this test class is categorized as functional on master?  From what I gauge based on the CI logs, this class is run under unit tests.
<img width="1662" alt="Screenshot 2025-06-09 at 18 07 07" src="https://github.com/user-attachments/assets/44bd9fbe-ba74-4175-9167-94cb21c28bca" />
<img width="1583" alt="Screenshot 2025-06-09 at 18 13 54" src="https://github.com/user-attachments/assets/3adb8152-1ddb-4374-be6c-a15724500254" />

Reply:
Similar on other `@Tag("functional")` usage

Reply:
I added the tag to the directories flagged by the `Suite` classes. For example, some tests under a `functional` directory are not currently tagged. Do we want to run them in the functional test suite or regular unit tests?

Reply:
Mentioned here: https://github.com/apache/hudi/pull/13407#pullrequestreview-2911882842 it's mainly for grouping tests in Azure to balance job completion time.  So I suggest keeping the existing tagging if that does not affect logging for your debugging.